src/main/java/net/ravendb/client/documents/DocumentStoreBase.java

```diff
@@ -146,6 +147,15 @@ public TimeSeriesOperations timeSeries() {
     }
 
     private DocumentConventions conventions;
+    private AiOperations aiOperations;
+
+    @Override
+    public AiOperations getAiOperations(){
```

Comment:
should not this be 
```suggestion
    public AiOperations AI(){
```
or
```suggestion
    public AiOperations ai(){
```

similar to timeseris, bulkinsert, subscriptions, etc?

Reply:
done